src/neptune/internal/operation_processors/operation_storage.py

```diff
@@ -34,29 +34,46 @@ def __init__(self, data_path: Union[str, Path]):
             self._data_path = data_path
         else:
             self._data_path = Path(data_path)
-
         # initialize directories
         os.makedirs(self.data_path, exist_ok=True)
         os.makedirs(self.upload_path, exist_ok=True)
 
+        cwd = Path(os.getcwd())
+        self._data_path_abs = cwd / self._data_path
+
     @property
     def data_path(self) -> Path:
         return self._data_path
 
+    @property
+    def data_path_abs(self) -> Path:
+        return self._data_path_abs
+
     @property
     def upload_path(self) -> Path:
         return self.data_path / "upload_path"
 
+    @property
+    def upload_path_abs(self) -> Path:
+        return self.data_path_abs / "upload_path"
+
     @staticmethod
     def _get_container_dir(type_dir: str, container_id: UniqueId, container_type: ContainerType):
         return f"{NEPTUNE_DATA_DIRECTORY}/{type_dir}/{container_type.create_dir_name(container_id)}"
 
     def close(self):
         shutil.rmtree(self.data_path, ignore_errors=True)
 
-        parent = self.data_path.parent
+        try:
+            parent = self.data_path.parent
+
+            files = os.listdir(parent)
+        except FileNotFoundError:
```

Comment:
Try to write a test for the behavior that is missing (failing) - the one with changeing the current working directory.